README.md

```diff
@@ -49,12 +49,12 @@
     To add the patch, add the two lines below to the spec file in the section for patches (usually right below the sources).
     ```
     # ACS override patch
-    Patch1000: add-acs-overrides.patch
+    Patch1000: add-acs-override.patch
     ```
 
-    Then tell it to apply the patch in the `prep` section.
+    Then tell it to apply the patch in the `prep` section. It will be below the `ApplyOptionalPatch()` variable, normally right above the `# END OF PATCH APPLICATIONS` note.
```

Comment:
```suggestion
    Then tell it to apply the patch in the `prep` section. It will be below the `ApplyOptionalPatch()` function definition, normally right above the `# END OF PATCH APPLICATIONS` comment.
```

I love it!  Does this make it even more clear?

Reply:
Sounds good, thanks!